src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannParser.cs

```diff
@@ -373,7 +373,7 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
             _logger.Debug($"Got {releases.Count} releases");
 
-            return releases;
+            return releases.OrderByDescending(o => o.PublishDate).ToArray();
```

Comment:
Some indexers have sort settings, does this make it confusing when results are not in same order?